process/eventsHandler_test.go

```diff
@@ -66,34 +66,45 @@ func TestHandlePushEvents(t *testing.T) {
 	t.Run("broadcast event was called", func(t *testing.T) {
 		t.Parallel()
 
+		events := data.BlockEvents{
+			Hash:      "hash1",
+			ShardID:   1,
+			TimeStamp: 1234,
+			Events:    []data.Event{},
+		}
+
 		wasCalled := false
 		args := createMockEventsHandlerArgs()
 		args.Publisher = &mocks.PublisherStub{
-			BroadcastCalled: func(events data.BlockEvents) {
+			BroadcastCalled: func(evs data.BlockEvents) {
+				require.Equal(t, events, evs)
 				wasCalled = true
 			},
 		}
 
 		eventsHandler, err := process.NewEventsHandler(args)
 		require.Nil(t, err)
 
-		events := data.BlockEvents{
-			Hash:   "hash1",
-			Events: []data.Event{},
-		}
-
 		eventsHandler.HandlePushEvents(events)
 		require.True(t, wasCalled)
 	})
 
 	t.Run("check duplicates enabled, should not process event", func(t *testing.T) {
 		t.Parallel()
 
+		events := data.BlockEvents{
+			Hash:      "hash1",
+			ShardID:   1,
+			TimeStamp: 1234,
+			Events:    []data.Event{},
+		}
+
 		wasCalled := false
 		args := createMockEventsHandlerArgs()
 		args.Config.CheckDuplicates = true
 		args.Publisher = &mocks.PublisherStub{
 			BroadcastCalled: func(events data.BlockEvents) {
+				require.Equal(t, events, events)
```

Comment:
can you fix this check?
events is always equal to itself
Also check the other tests as well.

Reply:
pushed for all the tests